pkg/api/v1beta1/dynakube/convert_from_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	"testing"
 
 	"github.com/Dynatrace/dynatrace-operator/pkg/api/status"
-	v1beta2 "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta2/dynakube"
+	v1beta2 "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
```

Comment:
```suggestion
	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
```

Reply:
fixed in 39ed827a

Reply:
github (and I) disagree 😅  (you fixed `pkg/api/v1beta2/dynakube/convert_from_test.go`)

Reply:
fixed 😅